src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -2269,4 +2269,39 @@ public Object executeScript(String script) {
         return getSeleniumHelper().executeJavascript(statement);
     }
 
+    /**
+     * @param place to check
+     * @return true if content of place is valid (according to validation pattern)
+     */
+    public boolean valueOfIsValid(String place) {
```

Comment:
Can you add some documentation and acceptance test for this new method?

Reply:
Ok